tilavarauspalvelu/migrations/0055_migrate_instructions_to_html.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 5.1.3 on 2024-12-16 23:51
+from __future__ import annotations
+
+import re
+
+from django.db import migrations
+
+
+def htmlize(text: str | None) -> str:
+    """Convert linebreaks in text to paragraphs."""
+    if not text:
+        return text
+
+    # Assume the text is already in HTML, skip conversion
+    if text.startswith("<"):
+        return text
+
+    text = re.sub(r"(.{1,})", r"<p>\1</p>", text)
```

Comment:
This is a one-off regex, so you can leave it, but I would also use more descriptive regex here.